packages/web/src/common/store/pages/collection/sagas.js

```diff
@@ -31,13 +31,15 @@ function* watchFetchCollection() {
         permalink,
         {
           requiresAllTracks: true,
-          deleteExistingEntry: true
+          deleteExistingEntry: true,
+          forceRetrieveFromSource: fetchLineup
```

Comment:
this means we'll be making a fresh call to discovery every time we hit a collection page. That's not necessarily wrong, but it may be possible to avoid it

Reply:
exactly that, not sure if this is the right move. i guess my main q is what is `fetchLineup` meant to do?

Reply:
I thought it was saying get the tracks not just the collection metadata. But I think that has cache hits as well so that's probably what's happening.

Not too crazy to force refresh here, but I suspect that's the difference if any.

Reply:
hmmm ok you're right actually. i think i should just add a separate flag.